src/UI/Implementation/Component/Icon/Standard.php

```diff
@@ -138,4 +144,22 @@ public function getAllStandardHandles() {
 		return self::$standard_icons;
 	}
 
+    /**
+     * @return bool
+     */
+    public function isOutlined(): bool
+    {
+        return $this->is_outlined;
+    }
+
+    /**
+     * @param bool $is_outlined
+     * @return Icon
+     */
+    public function withIsOutlined(bool $is_outlined)
+    {
+        $clone = clone $this;
+        $clone->is_outlined = $is_outlined;
+        return $clone;
+    }
```

Comment:
I think we are mixing indentations here. The spaces match our general decision to use PSR-2 some times in the future, but I think we should not mix in one file.